src/internal_events/splunk_hec.rs

```diff
@@ -44,3 +46,77 @@ impl InternalEvent for SplunkEventEncodeError {
         );
     }
 }
+
+#[cfg(feature = "sources-splunk_hec")]
+mod source {
+    use super::*;
+    use crate::sources::splunk_hec::ApiError;
+
+    #[derive(Debug)]
+    pub struct SplunkHECEventReceived;
+
+    impl InternalEvent for SplunkHECEventReceived {
+        fn emit_logs(&self) {
+            trace!(message = "received one event.");
+        }
+
+        fn emit_metrics(&self) {
+            counter!(
+                "events_processed", 1,
+                "component_kind" => "source",
+                "component_type" => "splunk_hec",
+            );
+        }
+    }
+
+    #[derive(Debug)]
+    pub struct SplunkHECRequestReceived<'a> {
+        pub path: &'a str,
+    }
+
+    impl<'a> InternalEvent for SplunkHECRequestReceived<'a> {
+        fn emit_logs(&self) {
+            debug!(message = "received one request.", %self.path, rate_limit_secs = 10);
+        }
+
+        fn emit_metrics(&self) {
+            counter!(
+                "request_received", 1,
+                "component_kind" => "source",
+                "component_type" => "splunk_hec",
+            );
+        }
+    }
+
+    #[derive(Debug)]
+    pub struct SplunkHECRequestBodyInvalid {
+        pub error: std::io::Error,
+    }
+
+    impl InternalEvent for SplunkHECRequestBodyInvalid {
+        fn emit_logs(&self) {
+            error!(message = "invalid request body.",%self.error, rate_limit_secs = 10);
```

Comment:
Looks like `rustfmt` missed formatting this.

Reply:
Indeed.